doc/sphinx/source/projects/adt7420_console/adt7420_console.rst

```diff
@@ -0,0 +1,105 @@
+ADT7420 Console Application
+---------------------------
+
+==================
+Supported Hardware
+==================
+
+**Supported Devices:**
+
+* `ADT7420 <https://www.analog.com/en/products/adt7420.html>`_ 
+* `ADT7320 <https://www.analog.com/en/products/adt7320.html>`_
+
+**Supported Evaluation Boards:**
+
+* `EV-TempSense-ARDZ <https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/EV-TempSense-ARDZ.html>`_ 
+
+**Supported Carrier Boards:**
+
+* `SDP-K1 With Mbed <https://os.mbed.com/platforms/SDP_K1/>`_
+
+============
+Introduction
+============
+
+The ADT74XX and ADT73XX are a family of high accuracy digital temperature sensors offering breakthrough performance over a wide industrial range. The EV-Tempsense-ARDZ Mbed support software can be used as a starting point for developing your own code for Analog Devices EV-TempSense-ARDZ board in your own environment utilizing the benefits of the Mbed platform. Analog Devices is an MBED Partner and develops code on the platform for multiple products.
+The firmware example comprises 3 layers of software (from top to bottom): Console Application Layer, Device No-OS Layer and Platform Drivers (Mbed-OS) layer.
+
+   .. image:: /source/projects/adt7420_console/adt7420_architecture.jpg
```

Comment:
Image file not read: Change image extension to png since the actual image attached is in png